src/QMCWaveFunctions/TrialWaveFunction.h

```diff
@@ -207,13 +202,13 @@ class TrialWaveFunction : public MPIObjectBase
                           TinyVector<ParticleSet::ParticleGradient_t, OHMMS_DIM>& grad_grad,
                           TinyVector<ParticleSet::ParticleLaplacian_t, OHMMS_DIM>& lapl_grad);
 
-  RealType ratioGrad(ParticleSet& P, int iat, GradType& grad_iat);
+  ValueType calcRatioGrad(ParticleSet& P, int iat, GradType& grad_iat);
```

Comment:
Probably need a documentation comment here.  What are the arguments & what's done to them (looking at grad_iat), and what's the return value.

Reply:
Since after this change, the naming at TWF and WFC is different. Documentation is definitely needed. I'm working on another PR adjust the accumulation and return precision of calcRatio and calcRatioGrad, I will get all the documentation up.

Reply:
@ye-luo please keen an eye for any missing unit test coverage while doing so.